unit_1d_hw.py

```diff
@@ -7,8 +7,180 @@
+
+#### Create a Function Below ####
+'''
+Create a function called string_return_I_am_a_string() that returns the string "I am a string!"
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called add_one(), that takes in a parameter x, adds 1 then returns the value
+
+'''
+
+##################################
+
+
+#### Create a Function Below ####
+'''
+Create a function called add_x_y, that takes in a parameter x, and a parameter y, adds x and y then returns that value
+
+'''
+
+##################################
+
+
+#### Create a Function Below ####
+'''
+Create a function called slope, that takes in parameters, m, x and b. Where:
+
+:param m: (float) The slope to scale the the list x
+:param x: (list) The list of values we want to scale and shift based on m and b
+:param b: (float) The intercept to shift the values of x
+
+Have the function scale each value of x then add b and store the result back into x.
+
+Have the function return x
+
+Hint: Use a for loop to iterate through the list
+'''
+
+
+##################################
+
+
+
+#### Create a Function Below ####
+'''
+Create a function called multi_return, that takes in parameters, x and y
+
+:param x: (int) Integer passed in
+:param y: (int) Integer passed in
+
+Have the function scale the of x by 5 and add 2 to y. 
+
+Have the function return x and y in that order. 
+
+'''
+
+##################################
+
+
+#### Create a Function Below ####
+'''
+Create a function called default_args, that takes in parameter pre_pend and x
+
+:param pre_pend: (int) Integer we would like to prepend to the list x
+:param x: (list) List we want to prepend onto
+:return x: (list) List after prepend
+
+Have the default value for pre_pend be 5. Have the function prepend the value pre_pend to the list x.
+
+Return the list x.
+
+
+'''
+
+##################################
+
+
+##### Create a Class Below #######
+'''
+Create a class called Car, create an init method (constructor) that takes in 
+miles_per_gallon, and assigns it to the attribute `mpg`.
+
+'''
+
+##################################
+
+##### Create a Class Below #######
+'''
+Create a class called Dog, create an init method (constructor) that takes in breed, and assigns
+it to the attribute `breed`. Have init also take in weight, an assign it to the attribute `weight`.
+
+Then create a method (aka class function) called `get_breed`. It should return the
+`breed` attribute. 
+
+Then create a method called `get_weight`. It should return the `weight` attribute.
+
+Note: The get_x functions should not have an input. Ex: def get_weight():
+
+'''
+
+##################################
+
+
+
+
+##### Create a Class Below #######
+'''
+Create a class called Math_Operations
+
+Create a method called add, that takes in a and b, adds them together (a+b) and returns the value. 
+
+Create a method called sub, that takes in a and b, then subtracts b from a (b-a), and returns the value.
```

Comment:
I think "subtracts b from a" would need to be (a-b)

```suggestion
Create a method called sub, that takes in a and b, then subtracts b from a (a-b), and returns the value.
```

Reply:
yes sorry also sleep deprived moment